Lambda-School-Labs/mydish-fe
# Feature/create recipe form for edit

This PR finally merges `fake-master` into `master`.  😉
But seriously, folks, this PR changes the editable recipe to use the same UI as creating a recipe. 
**To test:** 
1. Create a new recipe.
2. Try to break things while you do it.
3. Edit a saved recipe.
4. Try to break things while you do it.
5. Make sure all of your valid* edits are included in the new saved recipe.
6. Make sure the `Previous versions` list is accurate.

*An _invalid_ edit would be something like an instruction with a bunch of newline characters, or an empty ingredient. Those edits won't be saved.
